bank/src/main/java/org/example/Constant/ErrorCode.java

```diff
@@ -0,0 +1,34 @@
+package org.example.Constant;
+
+
+//에러코드 - using enum class
+public enum ErrorCode {
+
+    // Unexpected Error
+    E000("E000", "예상치 못한 오류 발생"),
+
+    // Custom Error
+    E101("E101", "출금 처리 중 에러 발생"),
+    E102("E102", "입금 처리 중 에러 발생"),
+    E103("E103", "송금 처리 중 에러 발생"),
+    E104("E104", "적금 계좌는 잔액이 목표 금액(%s원) 이상이어야 출금 가능합니다."),
+
+    // IO Error
+    E201("E201", "입력값 오류 발생");
```

Comment:
위 숫자들은 어떤 의미를 가지고 있나요? 더 의미있는 이름을 사용해보도록 합시다

Reply:
```suggestion
    INPUT_ERROR("E201", "입력값 오류 발생");
```
처럼 해당 필드의 제목은 떠오르기 쉽게 관리하는 것이 더 나을 것 같습니다!